docker/default_config.ini

```diff
@@ -70,6 +70,9 @@ rpc-endpoint = 0.0.0.0:8090
 # For database_api_impl::get_full_accounts to set max items to return in the lists
 # api-limit-get-full-accounts-lists = 500
 
+# For database_api_impl::get_top_voting_power_accounts to set max limit value
+# api-limit-get-top-n-voting-accounts = 200
```

Comment:
It would be confusing that the naming is inconsistent.

BTW I think the API name is a bit too long, `get_top_voting_accounts` or even `get_top_voters` might be better. What do you think?

Reply:
I think `get_top_voters` is better.